.gitignore

```diff
@@ -1,17 +1,51 @@
-/build
-library/build
-demo/build
-
-# Android generated
-bin
-gen
-lint.xml
-local.properties
+# Built application files
+*.apk
+*.ap_
+
+# Files for the Dalvik VM
+*.dex
 
-# IntelliJ IDEA
-.idea
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+.idea/
 *.iml
-*.ipr
```

Comment:
We should keep the *.ipr and *.iws for those that want to use that format for AndroidStudio projects.